dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -189,4 +189,6 @@ void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
     DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+
+    String buildFunctionCall(String procOrFuncName, int numberOfParameters);
```

Comment:
It's possible but there is some risk to end up with too deep refactoring again. It's actually possible to have smaller more focused interface for the part of interest now.

But what about just extending `DbStoredProcedureCall`? I think this way we can avoid the need for tweaking the environment API. So far this has been the way to customize behavior e.g. for Oracle.